syft/frameworks/torch/torch_attributes.py

```diff
@@ -86,8 +84,8 @@ def __init__(self, torch: ModuleType, hook: ModuleType) -> None:
             "is_storage",
             "is_tensor",
             "isclose",
-            "isfinite" "load",
-            "long",
+            "isfinite",
+            "load",
```

Comment:
why is long removed

Reply:
Because it doesn't need to be excluded I think